lib/glimesh/streams/title.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Glimesh.Streams.Title do
+  alias Glimesh.Repo
+  alias Glimesh.Streams.Channel
+  alias Glimesh.Events
+
+  def change_title(channel, title) do
```

Comment:
This should already be handled by `Glimesh.Streams.update_channel(streamer, channel, %{title: "some title"})`